### DISCUSSION TITLE
Add categories of related products

### DISCUSSION THREAD

**Discussion by yannicka** [PrestaShop/PrestaShop#40183] [Ideas]
PrestaShop offers a mecanism of "related products" that allows us to associate products to another.

This allows for example to list complementary products, similar products, products of substitution, etc.

However, this allows only one list of "related products", and it is not possible to know if a product is associated because it is complementary, similar, a substitution or whatever.

The idea: to be able to create categories of related products (the most standard categories would be created by default in PrestaShop) then to be able to associate a a related product to one (or even several?) category.

---

**For example:**

Categories:

ID  | Slug       | Name
--- | ---------- | -------------------------
 1  | crosssell  | Cross sell / Complementary products
 2  | similar    | Similar products
 3  | upsell     | Upsell products
 4  | substitute | Substitute products

Related products of "Product A":

Related product | Category slug
--------------- | -------------
Product B       | crosssell
Product C       | crosssell
Product D       | similar
Product E       | substitute
Product F       | substitute

Mock-up:
![maquette-ps](https://user-images.githubusercontent.com/22885898/73362938-0db31300-42a8-11ea-9bdd-b302fc88979c.png)

**Comment by khouloudbelguith** [comment]
Hi @yannicka,

Thanks for your report.
Ping @PrestaShop/prestashop-product-team what do you think?

Thanks!

**Comment by Hlavtox** [comment]
@khouloudbelguith I upvote for this. It would be very useful.

@yannicka You can make it by copying the accessories table and functionality.
